### ISSUE TITLE
NuGet Pack includes non-portable PDB content files in .snupkg

### ISSUE THREAD

**Issue by dauthleikr** [NuGet/Home#12774]
### Impact

I'm unable to use NuGet.org

### Describe the bug

I am not able to upload a generated snupkg to Nuget.org. During indexing it always reports this error "The uploaded symbols package contains one or more pdbs that are not portable."

The snupkg in question contains a .NET assembly with a portable PDB (file starts with "BSJB"), but also a C++ component and its pdb. I suspect the portability-validation fails for the C++ pdb (starting with `Microsoft C/C++ MSF 7.00`), but there is nothing I can do here, since as far as I know, the portable-pdb format is a concept for .NET assemblies only.

### Repro Steps

1. Create snupkg with valid pdb from .NET assembly, and PDB from generated by MSVC in contentFiles
2. Try uploading to nuget.org

### Expected Behavior

Do not validate C++ pdbs

**Comment by dauthleikr** [comment]
Is the correct way here to just not ship the C++ pdb with the snupkg, and only deliver it via contentFiles in the nupkg? Annoyingly it is automatically included when specifying `<IncludeSymbols>True</IncludeSymbols>` in the csproj ...

I have also temporarily uploaded the package for you to inspect:
[https://honeyflare.neuwirth.dev/Grind.2023.7.14.1.nupkg](https://honeyflare.neuwirth.dev/Grind.2023.7.14.1.nupkg)
[https://honeyflare.neuwirth.dev/Grind.2023.7.14.1.snupkg](https://honeyflare.neuwirth.dev/Grind.2023.7.14.1.snupkg)

**Comment by joelverhagen** [comment]
I think your theory about it failing on the C++ PDB. The NuGet.org symbol server only accepts so-called "portable" PDBs. We can't change this requirement for security reasons. However, excluding that particular PDB sounds like a setting you would need to put in the project file (some kind of setting for NuGet client code that does `pack` to read).

I'll pull in some NuGet client folks to see if there's a way to exclude some PDBs (`Grind.LowLevel.pdb`) from the .snupkg. Frankly, it's strange to me that content files are included in the PDB at all. I would expect just .pdbs in runtime top level directories like `lib`.

**Comment by dauthleikr** [comment]
@joelverhagen Yes, excluding PDB's is what I'd need to make a clean package without some trickery. I guess even changing the default behavior so that only PDB's are packed which can be used by the NuGet symbol server could also be sensible.

**Comment by joelverhagen** [comment]
It looks like we have two open questions then:

1. Is it expected for NuGet client to pack non-portable PDBs into the content files of a .snupkg? (I think the answer would be know because .snupkg was specifically designed for NuGet.org and the portable PDBs requirement).
2. How to work around the problem by excluding these PDB content files from the resulting .snupkg?

In light of this, I will move the issue to NuGet/Home since this appears to be more of a client issue than a NuGet.org issue. NuGet.org is doing what is expected: rejecting non-portable PDBs in the .snupkg.
